lib/site_template/_config.yml

```diff
@@ -29,6 +29,15 @@ markdown: kramdown
 theme: minima
 gems:
   - jekyll-feed
-exclude:
-  - Gemfile
-  - Gemfile.lock
+
+# Exclude from processing.
+# The following items will not be processed, by default. Create a custom list
+# to override the default setting.
+# exclude:
```

Comment:
we'll probably have to modify the `deep_merge_hashes` utility method to `append!` when `hash[key].is_a? Array` ...